lms/templates/register-form.html

```diff
@@ -88,7 +93,7 @@ <h2 class="sr">${_('Required Information')}</h2>
     <li class="field required text" id="field-username">
       <label for="username">${_('Public Username')}</label>
       <input id="username" type="text" name="username" value="${username}" placeholder="${_('example: JaneDoe')}" required aria-required="true" aria-describedby="username-tip"/>
-      <span class="tip tip-input" id="username-tip">${_('Will be shown in any discussions or forums you participate in')} <strong>(${_('cannot be changed later')})</strong></span>
+      <span class="tip tip-input" id="username-tip">${_('Will be shown in any discussions or forums you participate in')} <strong>(${Text(_('cannot be changed later'))})</strong></span>
```

Comment:
Nit: Does seem like `Text` is needed here.  Not sure why the linter would have complained.  It also shouldn't break anything though.